From 351d1c37a525958e6dd0da551abb7a82e1aeb173 Mon Sep 17 00:00:00 2001
From: rajucm <rajkumar.mallikarjun.chegaraddi@intel.com>
Date: Thu, 15 Mar 2018 23:59:40 +0530
Subject: [PATCH] touchscreen: fix multiple touchscreen device detected for
 same hardware.

add fix in input device capablity file to avoid creating
an extra touch device for the same hardware.

Change-Id: I25dd460279ece31640eea4ad8bd10953d691a756
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-59133
Signed-off-by: rajucm <raju.mallikarjun.chegaraddi@intel.com>
---
 touch/Vendor_0eef_Product_7200.idc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/touch/Vendor_0eef_Product_7200.idc b/touch/Vendor_0eef_Product_7200.idc
index b952c9b..5404be6 100644
--- a/touch/Vendor_0eef_Product_7200.idc
+++ b/touch/Vendor_0eef_Product_7200.idc
@@ -1,2 +1,2 @@
 device.internal = 1
-touch.deviceType = touchScreen
+# touch.deviceType = touchScreen
-- 
1.9.1

